src/Raven.Server/Documents/Expiration/ExpiredDocumentsCleaner.cs

```diff
@@ -119,15 +119,15 @@ private async Task DoExpirationWork()
 
         internal Task CleanupExpiredDocs(int? batchSize = null)
         {
-            return CleanupDocs(batchSize ?? BatchSize, forExpiration: true);
+            return CleanupDocs(batchSize ?? BatchSize, ExpirationConfiguration.ExpirationMaxItemsToProcess ?? BatchSize, forExpiration: true);
```

Comment:
We want to keep deleting the expired/refreshed docs in a `while(true)` loop by default, but have an ability to set the `ExpirationMaxItemsToProcess` configuration on low tier machines. 
So lets make `ExpirationConfiguration.ExpirationMaxItemsToProcess` null or `int.MaxValue` by default for x64 bits (so it will remove everything without waiting on each iteration), and maybe we can limit it for 32bits (maybe `BatchSize` x 16 so it will make 16 iterations, 65,536 total docs)

Reply:
done